lib/bq_stream/bq_stream.rb

```diff
@@ -91,6 +91,16 @@ def dequeue_items
       log(:info, "#{Time.now}: ***** Dequeue Items Ended ***** #{log_code}")
     end
 
+    def insert_missing_records(records, bq_attributes)
+      bq_attributes.each do |bqa|
+        records.each do |record|
+          unless record.send(bqa).nil?
+            QueuedItem.create(table_name: record.class.to_s, record_id: record.id, attr: bqa.to_s, new_value: record.send(bqa), updated_at: record.updated_at)
+          end
+        end
+      end
+    end
```

Comment:
This only works if you know every single missing record/attribute right?

Reply:
Correct, using the known missing ids and then grabbing the bq_attributes from the model you can get them into bigquery